applications/TopologyOptimizationApplication/tests/Small_Cantilever/OptimizationParameters.py

```diff
@@ -0,0 +1,89 @@
+#========================================================================================
+# DESIGN VARIABLES
+#========================================================================================
+
+optimization_method = "simp_method" 
```

Comment:
Parameters are a json, not a python file (that was the old style, see the StructuralMechanicsApplication for example)